### PR TITLE
Feature/29 add modal to reporting for more info

### LINKED ISSUE #29: Add modal to reporting for more info

### PR DESCRIPTION
**Pull request by RabbITCybErSeC**

### CODE REVIEW

File: utils/errors.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package utils
```
Comment (MaartendeKruijf) [inline utils/errors.go:1]: Just use error.Join,error.Unwrap and error.Is https://pkg.go.dev/errors#Join

File: views/components/alerts/alerts.templ [hunk 2]
```diff
@@ -0,0 +1,63 @@
+package alerts
+
+import "soarca-gui/utils"
+
+const baseClassWarning = "flex w-full max-w-sm overflow-hidden bg-white rounded-lg shadow-md dark:bg-gray-800"
```
Comment (MaartendeKruijf) [inline views/components/alerts/alerts.templ]: Upper CammelCase for consts

**Review by MaartendeKruijf — CHANGES_REQUESTED** [review]

File: views/components/alerts/alerts.templ (hunk 2 above)
Comment (RabbITCybErSeC) [reply views/components/alerts/alerts.templ]: why there is no formality stating that they should be public? this is just a private const?

**Review by MaartendeKruijf — CHANGES_REQUESTED** [review]
Update the backend error handling the rest looks good

**Review by MaartendeKruijf — APPROVED** [review]

**Pull request merged**